### DISCUSSION TITLE
Problem in the 'Documents' section

### DISCUSSION THREAD

**Discussion by dauzeine** [chamilo/chamilo-lms#5227] [General]
Hi,
since the upgrade to 1.11.26 (I know, it's been a while), our platform is experiencing difficulties in the document section.
Some of the folders show '0B' in size, even if there are thousands of files in it. It's not a blocking problem, but annoying one.
Worse, when you're in a folder, half (or so) of the PDF files won't show up, in ViewerJs : just a blank page. We need to download the file to see it.
I don't think it's a bug in chamilo, as I don't find anyone reporting this. I think of a config problem, in apache2 or so.
My config : Ubuntu server LTS, apache2, mysql8, php7.4 with all the modules needed.

I appreciate any help, idea... as usual!
david

**Comment by NicoDucou** [comment]
Hi @dauzeine 
Just to indicate that in 1.11.26 the calculation of the size of files and folders in the document tool as changed. To enable a quicker opening of the page now we show the list and then we load the size by ajax so the calculation can be done while the user already see it's list of content.
As you indicate a problem with ViewerJs it also seems to be related to ajax problems.
Do you have any errors in the javascript console of the browser that might help you understand what goes wrong ?
I hope this helps you find a bit more information about the reason of this problem.
Best regards

**Reply by dauzeine** [reply]
Hi @NicoDucou
thanks for your quick reply. I had indeed a problem with ajax, whatsoever the navigator (chromium, firefox), with GET.
Your answer helped me find that Headers, in particuliar script_src, that caused the problem. Disabling the script_src part in my VirtualHost did the trick.
Thank you so much!
Best regards
david
